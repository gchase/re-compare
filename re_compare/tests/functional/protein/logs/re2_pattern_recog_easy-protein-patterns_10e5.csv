regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,0.5440711975097656,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,0.5440711975097656,"[-1, -1]"
DHKNLD.,first_match,0,0.13518333435058594,"[-1, -1]"
DHKNLD.,all_matches,0,0.13518333435058594,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,0.6740093231201172,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,0.6740093231201172,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,0.015020370483398438,"[99268, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,0.6890296936035156,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,0.15997886657714844,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,0.15997886657714844,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.08487701416015625,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.08487701416015625,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.010251998901367188,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.09512901306152344,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,0.3859996795654297,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,0.3859996795654297,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,0.010967254638671875,"[50386, 50391]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,0.39696693420410156,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.17118453979492188,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.17118453979492188,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.014066696166992188,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.18525123596191406,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,0.19812583923339844,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,0.19812583923339844,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,0.286102294921875,"[58929, 58933]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,0.48422813415527344,"[-1, -1]"
P..L...KWE.,first_match,0,0.2727508544921875,"[-1, -1]"
P..L...KWE.,all_matches,0,0.2727508544921875,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,0.4260540008544922,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,0.4260540008544922,"[-1, -1]"
