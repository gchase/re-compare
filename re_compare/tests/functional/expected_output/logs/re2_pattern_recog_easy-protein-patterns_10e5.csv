regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,0.5590915679931641,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,0.5590915679931641,"[-1, -1]"
DHKNLD.,first_match,0,0.1461505889892578,"[-1, -1]"
DHKNLD.,all_matches,0,0.1461505889892578,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,0.6201267242431641,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,0.6201267242431641,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,0.012159347534179688,"[99268, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,0.6322860717773438,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,0,0.15401840209960938,"[-1, -1]"
(GI)(G|R)P.Y..K..,all_matches,0,0.15401840209960938,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.05793571472167969,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.05793571472167969,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0057220458984375,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.06365776062011719,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,0.39196014404296875,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,0.39196014404296875,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,0.010967254638671875,"[50386, 50391]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,0.4029273986816406,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.24580955505371094,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.24580955505371094,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.030040740966796875,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.2758502960205078,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,0.1971721649169922,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,0.1971721649169922,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,0.19288063049316406,"[58929, 58933]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,0.39005279541015625,"[-1, -1]"
P..L...KWE.,first_match,0,0.19788742065429688,"[-1, -1]"
P..L...KWE.,all_matches,0,0.19788742065429688,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,0.8370876312255859,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,0.8370876312255859,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),first_match,1,0.12302398681640625,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,1,0.12302398681640625,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,2,0.010967254638671875,"[996, 1004]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),all_matches,2,0.13399124145507812,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),first_match,1,0.5540847778320312,"[41351, 41357]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),consecutive,1,0.5540847778320312,"[41351, 41357]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),all_matches,1,0.8897781372070312,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),first_match,0,0.23603439331054688,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),all_matches,0,0.23603439331054688,"[-1, -1]"
A|G....G,first_match,1,0.2689361572265625,"[15, 15]"
A|G....G,consecutive,1,0.2689361572265625,"[15, 15]"
A|G....G,consecutive,2,0.009298324584960938,"[19, 19]"
A|G....G,all_matches,2,0.27823448181152344,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,first_match,1,0.23603439331054688,"[57782, 57786]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,1,0.23603439331054688,"[57782, 57786]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,2,0.009059906005859375,"[58001, 58005]"
((CP|.)(S|T)N)(I|L|V)G,all_matches,2,0.24509429931640625,"[-1, -1]"
